OutOfSchool/OutOfSchool.Admin/MediatR/Applications/Handlers/GetByFilterAllApplicationsHandler.cs

```diff
@@ -0,0 +1,21 @@
+﻿using MediatR;
+using OutOfSchool.Admin.MediatR.Applications.Queries;
+using OutOfSchool.BusinessLogic.Models;
+using OutOfSchool.BusinessLogic.Models.Application;
+using OutOfSchool.BusinessLogic.Services;
+
+namespace OutOfSchool.Admin.MediatR.Applications.Handlers;
+public class GetByFilterAllApplicationsHandler(ISensitiveApplicationService applicationService)
+    : IRequestHandler<GetByFilterAllApplicationsQuery, SearchResult<ApplicationDto>>
+{
+    private readonly ISensitiveApplicationService applicationService = applicationService;
+
+    public async Task<SearchResult<ApplicationDto>> Handle(GetByFilterAllApplicationsQuery request, CancellationToken cancellationToken)
+    {
+        var filter = request.Filter;
+
+        var applications = await applicationService.GetAll(filter).ConfigureAwait(false);
+
+        return applications;
+    }
+}
```

Comment:
(Generic comment about all handlers) At the moment I see that all handlers could be rewritten in next (much shorter) way:
```
public class CommandOrQueryHandler(IService service)
    : IRequestHandler<GetByFilterAllApplicationsQuery, SearchResult<ApplicationDto>>
{
    public Task<Result> Handle(CommandOrQuery request, CancellationToken cancellationToken)
        => service.MagicMethod(request.Param1, request.Param2...);
}
```
So, you don't need that extra `private readonly` field for service (like `applicationService` in your code), those local variables in `Handle()` method (like `filter`), and `async`/`await` for `Handle()` that (partially) hurts performance